statsmodels/tsa/vector_ar/svar_model.py

```diff
@@ -465,18 +469,19 @@ class SVARProcess(VARProcess):
     -------
     **Attributes**:
     """
-    def __init__(self, coefs, intercept, sigma_u, A_solve, B_solve,
+    @deprecate_kwarg('sigma_u', 'cov_resid')
+    def __init__(self, coefs, intercept, cov_resid, A_solve, B_solve,
                  names=None):
         self.k_ar = len(coefs)
         self.neqs = coefs.shape[1]
         self.coefs = coefs
         self.intercept = intercept
-        self.sigma_u = sigma_u
+        self.cov_resid = cov_resid
         self.A_solve = A_solve
         self.B_solve = B_solve
         self.names = names
 
-    def orth_ma_rep(self, maxn=10, P=None):
+    def orth_ma_rep(self, maxn=10, p=None):
```

Comment:
`svar_ma_rep` below also change `P`-->`p`?

Actually, why this change?  `P` is a matrix right?

Reply:
`P` is an ideal python variable name, `p` is.